### ISSUE TITLE
Adobe does not open when I click on a pdf file.

### ISSUE THREAD

**Issue by sb356607** [nvaccess/nvda#13351]
When I click on a pdf within NVDA it tries to open it and then it closes.You have arrived at a default issue template (bug report).
It is preferable to [choose the type of issue](https://github.com/nvaccess/nvda/issues/new/choose) to submit.
Hint: Swap to the preview tab to be able to click the links.
Direct links to the templates:
- [Bug report](https://github.com/nvaccess/nvda/issues/new?template=bug_report.md)
- [Feature request](https://github.com/nvaccess/nvda/issues/new?template=feature_request.md)

**Comment by TheQuinbox** [comment]
This doesn't sound like an NVDA Problem, it sounds like an Adobe problem. Maybe I'm missing something though (you didn't fill out the issue template).

**Comment by seanbudd** [comment]
Please fill out the issue template in future.
I am closing as this is likely a duplicate of #12920, and needs more information otherwise.

**Comment by sb356607** [comment]
Hello there Quinn, This is the first time I have done a report for NVDA.  I will also take a peak at the other bug report and see if I can add more to it.   Sent from Mail for Windows From: QuinSent: Wednesday, February 16, 2022 3:16 PMTo: nvaccess/nvdaCc: sb356607; AuthorSubject: Re: [nvaccess/nvda] Adobe does not open when I click on a pdf file. (Issue #13351) This doesn't sound like an NVDA Problem, it sounds like an Adobe problem. Maybe I'm missing something though (you didn't fill out the issue template).—Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by Brian1Gaff** [comment]
I have to say that in most caes, I've changed the default program for pdf 
opening to Microsoft Edge. It nearly always makes a better job of reading it 
than Adobe reader does.

Brian

***@***.***
Sent via blueyonder.
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----
